protostar/commands/test/test_environment_exceptions.py

```diff
@@ -81,6 +85,12 @@ class SimpleBreakingReportedException(
     pass
 
 
+class SimpleSkippingReportedException(
```

Comment:
Why 2 subclasses instead of 1?

Reply:
On top of this, why not simply one class `TestSkipped` which has reason out of the box?

Also, I don't think this has to be a reported exception? `HypothesisFailureSmugglingError` is not for example.

Reply:
Done